neuro_extras/image_builder.py

```diff
@@ -0,0 +1,193 @@
+import asyncio
```

Comment:
let's have a clear high-level separation by file `image`/`data`? IMO having this instead of `image.py` + `image_builder.py` and `data.py` + `data_copier.py` is easier to understand, even if some files will be large.

Reply:
ok

Reply:
Well, no. `ImageBuilder` is used by `image.py`, `config.py`, `k8s.py` and `seldon.py`.
`DataCopier` is used by `data.py` only, but it uses `NEURO_EXTRAS_IMAGE`, which is also used by `seldon.py`.

If we merge `ImageBuilder` and `DataCopier` to `image.py` and `data.py` respectively, we add interdependencies between modules, which I don't think is benefical (a first step to circular dependencies).

What do you think though?

Reply:
I'd extract common parts like `NEURO_EXTRAS_IMAGE` to a separate file like `common.py` or `constants.py`

Reply:
Ok, this allows us to merge `DataCopier` with `data.py`. How about `ImageBuilder` class which is used in 4 places? Shall we keep it separate?

Reply:
Well does `ImageBuilder` really needs to be used in 4 places? maybe we could refactor this? if it's a bad time, let's keep it in a separate file `utils.py`

Reply:
So your suggestion is essentially to rename `image_builder.py` to `utils.py`? Just making sure I understand your idea.